packages/remix-eslint-config/package.json

```diff
@@ -31,11 +31,9 @@
     "eslint-plugin-testing-library": "^5.0.5"
   },
   "peerDependencies": {
-    "@remix-run/react": ">=1.0.0",
     "eslint": ">=8.1.0",
     "react": ">=17.0.0",
-    "react-dom": ">=17.0.0",
-    "remix": ">=0.20.1"
```

Comment:
Removing internal peer deps as they cause all kinds of problems for pre-release tags